backend/app/api/endpoints/admin/device_monitor.py

```
@@ -0,0 +1,282 @@
+        db: Database session
+        current_user: Must be admin
+
+    Returns:
+        AdminDeviceListResponse with all devices matching filters
+    """
+    # Get all Device CRDs from the database
+    query = db.query(Kind).filter(
+        and_(
+            Kind.kind == "Device",
+            Kind.namespace == "default",
+            Kind.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)